scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hvssop.yaml

```diff
@@ -0,0 +1,55 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'HVSSOP'
+
+HVSSOP-10-1EP_3x3mm_P0.5mm_EP1.57x1.88mm:
+  size_source: 'https://www.ti.com/lit/ds/symlink/bq24090.pdf'
+
+  body_size_x:
+    nominal: 3
+    tolerance: 0.100
+  body_size_y:
+    nominal: 3
+    tolerance: 0.100
+  overall_height:
+    maximum: 1.1
+
+  overall_size_x:
+    nominal: 4.9
+    tolerance: 0.152
+  lead_width:
+    minimum: 0.17
+    maximum: 0.27
+  lead_len:
+    nominal: 0.53
+    tolerance: 0.152
```

Comment:
Where did you get that value from?

Reply:
This one was from the MSOP-10 (MSOP-10-1EP_3x3mm_P0.5mm_EP1.68x1.88mm_ThermalVias)

Reply:
I found `0.4 .. 0.7` in the datasheet.

Reply:
Updated